pkg/contracts/ton/gateway_test.go

```diff
@@ -262,6 +387,41 @@ func TestSnakeData(t *testing.T) {
 	}
 }
 
+func TestDeployment(t *testing.T) {
+	// ARRANGE
+	// Given TSS address & Authority address
+	const (
+		sampleTSSPrivateKey = "0xb984cd65727cfd03081fc7bf33bf5c208bca697ce16139b5ded275887e81395a"
```

Comment:
This can maybe moved to the testutil/samples package

Reply:
Let's keep it here and not rely on hard-coded stuff project-wise